packages/smart-contracts/src/contracts/ChainlinkConversionPath.sol

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.25 <0.7.0;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "openzeppelin-solidity/contracts/access/roles/WhitelistAdminRole.sol";
+
+interface ERC20fraction {
+  function decimals() external view returns (uint8);
+}
+
+interface AggregatorFraction {
+  function decimals() external view returns (uint8);
+  function latestAnswer() external view returns (int256);
+  function latestTimestamp() external view returns (uint256);
+}
+
+
+/**
+ * @title ChainlinkConversionPath
```

Comment:
I'm not sure what exactly we mean by conversion, but it's too generic to be clear. Whould it be clearer if we call it `rate`, `exchange rate` or `conversion rate`?

Reply:
my understanding is that this smart contract is no longer about converting rates, but providing the value, which btw, could be an issue: we will probably want to display the rate on the UI